### PR TITLE
Fix rendering for languages with only 1 plural

### PR DESCRIPTION
**Pull request by tomasr8**

### CODE REVIEW

File: src/client/index.js [hunk 1]
```diff
@@ -202,8 +202,27 @@ const getContextParams = args => {
   }
 };
 
+const getNPlurals = (...args) => {
+  // Only true in tests
+  if (args.length !== 1) {
+    return null;
+  }
```
Comment (ThiefMaster) [inline src/client/index.js]: why is that needed?

File: test-data/example.jsx [hunk 2]
```diff
@@ -30,7 +30,7 @@ const pgettext = (ctx, msg) =>
 const npgettext = (ctx, msg, msgpl, n) =>
   TRANSLATE ? jedTranslated.npgettext(ctx, msg, msgpl, n) : jedEmpty.npgettext(ctx, msg, msgpl, n);
 
-const {Translate, PluralTranslate} = makeComponents({gettext, ngettext, pgettext, npgettext});
+const {Translate, PluralTranslate} = makeComponents(gettext, ngettext, pgettext, npgettext);
```
Comment (ThiefMaster) [inline test-data/example.jsx]: how does this not break things? the argument we pass there (also when [using it in indico](https://github.com/indico/indico/blob/f0765e717cf16283a8ae3ba2705a8699276c6aad/indico/web/client/js/react/i18n.js#L11)) is the `jed` instance, which has those functions as methods.

Comment (tomasr8) [reply test-data/example.jsx]: So the method signature is actually `export const makeComponents = (...args) => {`
and there's this whole logic in `getGettextFuncs` to handle different arguments. You can either pass a single object with the functions as properties or you can pass the functions individually (though this is only used in the tests)

File: src/client/index.js (hunk 1 above)
Comment (tomasr8) [reply src/client/index.js]: Same reason as below. In the tests, we have e.g.
```js
const {Translate, PluralTranslate} = makeComponents(
    msg => msg,
    (msg, msgpl, n) => (n === 1 ? msg : msgpl)
);
```

File: test-data/example.jsx (hunk 2 above)
Comment (ThiefMaster) [reply test-data/example.jsx]: huh, i wonder what i was thinking when doing that back then :D

Comment (tomasr8) [reply test-data/example.jsx]: Well we _could_ simplify the signature and only allow a single object. That would just require a couple small changes to the tests where we call `makeComponents`

Comment (ThiefMaster) [reply test-data/example.jsx]: hmm maybe better if we only want to support jed anyway (and then in tests we simply pass something that looks enough like jed for this library to work)

Comment (tomasr8) [reply test-data/example.jsx]: I updated it to only take a single argument (the jed instance)
